<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>500</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Yu Gothic UI Light</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Ввод информации</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(50, 50, 50, 255), stop:1 rgba(70, 70, 70, 255));
</string>
  </property>
  <widget class="QPushButton" name="pb_OK">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>360</y>
     <width>200</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri Light</family>
     <pointsize>25</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 20;
color: rgb(190, 190, 190);
background-color: rgb(57, 57, 57);
border: 2px solid;
border-color: rgb(190, 190, 190);</string>
   </property>
   <property name="text">
    <string>ОК</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>90</y>
     <width>331</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic UI Light</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(190, 190, 190);
background-color: rgba(255, 255, 255, 0);
</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;Введите IP-адрес*:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>150</y>
     <width>331</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic UI Light</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(190, 190, 190);
background-color: rgba(255, 255, 255, 0);
</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; color:#bebebe;&quot;&gt;Введите количество пакетов:&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; color:#ababab;&quot;&gt;(по умолчанию 4 пакета)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>230</y>
     <width>331</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic UI Light</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(190, 190, 190);
background-color: rgba(255, 255, 255, 0);
</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; color:#bebebe;&quot;&gt;Введите длительность таймаута:&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; color:#ababab;&quot;&gt;(по умолчанию 4 секунды)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>455</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(171, 171, 171);color: rgb(190, 190, 190);
background-color: rgba(255, 255, 255, 0);
</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Обязательно к заполнению*&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_ip">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>93</y>
     <width>201</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic UI Light</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 8;
color: rgb(190, 190, 190);
background-color: rgb(57, 57, 57);
border: 2px solid;
border-color: rgb(190, 190, 190);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_packets">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>158</y>
     <width>201</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic UI Light</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 8;
color: rgb(190, 190, 190);
background-color: rgb(57, 57, 57);
border: 2px solid;
border-color: rgb(190, 190, 190);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_timeout">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>238</y>
     <width>201</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic UI Light</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 8;
color: rgb(190, 190, 190);
background-color: rgb(57, 57, 57);
border: 2px solid;
border-color: rgb(190, 190, 190);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QRadioButton" name="rb_range">
   <property name="geometry">
    <rect>
     <x>371</x>
     <y>130</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(190, 190, 190);
background-color: rgba(255, 255, 255, 0);
</string>
   </property>
   <property name="text">
    <string>Диапазон IP-адресов</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rb_ip">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>130</y>
     <width>82</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(190, 190, 190);
background-color: rgba(255, 255, 255, 0);
</string>
   </property>
   <property name="text">
    <string>IP-адрес</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rb_link">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>130</y>
     <width>82</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(190, 190, 190);
background-color: rgba(255, 255, 255, 0);
</string>
   </property>
   <property name="text">
    <string>Ссылка</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
